spec/features/user_logs_out_spec.rb

```diff
@@ -20,14 +20,14 @@
 
     click_button 'Log in'
 
-    expect(page).to have_content('Signed in successfully')
+    expect(page).to have_content('Logged in successfully')
 
-    click_link 'Sign Out'
-    expect(page).to have_content('Signed out successfully')
+    click_link 'Log Out'
+    expect(page).to have_content('Logged out successfully')
   end
 
   scenario 'unauthenticated user attempts to sign out' do
     visit '/'
-    expect(page).to_not have_content('Sign Out')
+    expect(page).to_not have_content('Log Out')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.